tests/Tests/200__monitor_and_manage/200__metrics/200__metrics.robot

```diff
@@ -1,26 +1,134 @@
 *** Settings ***
 Resource    ../../../Resources/ODS.robot
 
+
 *** Variables ***
+@{RECORD_GROUPS}    Availability Metrics    SLOs - JupyterHub    SLOs - ODH Dashboard    SLOs - RHODS Operator    SLOs - Traefik Proxy    Usage Metrics
 
-@{RECORD_GROUPS}  Availability Metrics  SLOs - JupyterHub  SLOs - ODH Dashboard  SLOs - RHODS Operator  SLOs - Traefik Proxy  Usage Metrics
+@{ALERT_GROUPS}     Builds                  DeadManSnitch    RHODS-PVC-Usage    SLOs-haproxy_backend_http_responses_total    SLOs-probe_success
 
-@{ALERT_GROUPS}  Builds  DeadManSnitch  RHODS-PVC-Usage  SLOs-haproxy_backend_http_responses_total  SLOs-probe_success
 
 *** Test Cases ***
-
 Test Existence of Prometheus Alerting Rules
-  [Tags]  Smoke  Sanity  ODS-509
-  Check Prometheus Alerting Rules
+    [Tags]    Smoke
+    ...       Sanity
+    ...       ODS-509
+    Check Prometheus Alerting Rules
 
 Test Existence of Prometheus Recording Rules
-  [Tags]  Smoke  Sanity  ODS-510
-  Check Prometheus Recording Rules
+    [Tags]    Smoke
+    ...       Sanity
+    ...       ODS-510
+    Check Prometheus Recording Rules
 
-*** Keywords ***
+Test Metric "Notebook CPU Usage" On ODS Prometheus
+    [Documentation]    Verifing the notebook cpu usage showing on RHODS promethues
+    [Tags]    Sanity
+    ...       Tier1
+    ...       ODS-178
+    Set Library Search Order    SeleniumLibrary
+    @{cur_usage} =    Read Current CPU Usage
```

Comment:
Could it be just a plain scalar variable (not a list)?  Also, can you rename it to ${cpu_usage_before} ?